zomes/hc_zome_trust_atom/src/lib.rs

```diff
@@ -11,40 +11,53 @@
 // #![warn(clippy::cargo)]
 
 use hdk::prelude::*;
+use std::collections::BTreeMap;
 
 // public for sweettest; TODO can we fix this:
 pub mod trust_atom;
 pub use crate::trust_atom::*;
-
 pub mod test_helpers;
-// pub use crate::test_helpers;
+pub use crate::test_helpers::*;
```

Comment:
Where do we use this?  Seems suspicious. 😎

Reply:
the test_helper functions at the bottom